package.json

```diff
@@ -72,7 +72,7 @@
         "localize-dist": "node scripts/properties2js dist",
         "unlocalize": "rimraf src/nls && git checkout -- src/nls",
         "postinstall": "grunt install",
-        "build": "grunt build-browser",
+        "build": "grunt build-browser && grunt watch:bramble",
```

Comment:
actually, I think we might just have incorrect documentation for this.
Why don't we just set `build` to be `grunt build-browser-dev`,
add another script called `"watch:api"` that runs `grunt watch:bramble`
install a new package called `npm-run-all`
and change `start` to be `run-p server watch:api || true`

That way, you basically never have to run `npm run build` and only always run `npm start` which will do the file watching for you.